lib/enigma.rb

```diff
@@ -21,4 +21,12 @@ def decrypt(message, key = self.default_key, date = self.default_date)
       date: shift[2]
     }
   end
+
+  def crack(message, date = self.default_date)
+    cracked_message_key = crack_message(message, date)
+    { decryption: decrypt_message(message, cracked_message_key[0]),
+      key: cracked_message_key[1],
+      date: cracked_message_key[2]
+    }
```

Comment:
crack in enigma calls on crack_message to find the shift value and then runs decrypt_message with the found key